Support page preview settings

Aced14 opened the pull request:
**Screenshots...**

**Settings...**

**Default:**
![page-preview-default](https://user-images.githubusercontent.com/13129485/56461493-1b835180-6382-11e9-8fb3-755dee0959ed.png)

**Before:**
![page-preview-before](https://user-images.githubusercontent.com/13129485/56461496-1faf6f00-6382-11e9-8855-916d8304c4b3.png)

**After:**
![page-preview-after-v2](https://user-images.githubusercontent.com/13129485/56708138-a600e380-66e9-11e9-8315-e8c33c161b4d.png)

**Disable notice...**

**Default:**
![page-preview-close-default](https://user-images.githubusercontent.com/13129485/56709138-b1eea480-66ed-11e9-83a4-526a50f8ea99.png)

**Before:**
![page-preview-close-before](https://user-images.githubusercontent.com/13129485/56709139-b31fd180-66ed-11e9-9499-542c25a437ab.png)

**After:**
![page-preview-close-after](https://user-images.githubusercontent.com/13129485/56709141-b3b86800-66ed-11e9-8314-72b2ed3c8021.png)

Aced14 commented:
Just force-pushed an update with a few corrections:
* Moved new selectors the end of pre-existing selector groups
* Ensured modified lines don't exceed 80 characters
* Replaced a random tab character with 4 spaces

Aced14 commented:
@the-j0k3r @Mottie Any thoughts on **Page preview/Enable/Disable**'s font colour?

In Wikipedia's default theme, it's ``#000000``. It's currently ``#9a9a9a`` in this PR (i.e. this theme's normal font colour). I could change it to ``#ffffff`` if you'd prefer.

the-j0k3r commented on wikipedia-dark.user.css:
```diff
@@ -447,7 +450,8 @@ domain("wiki.rpcs3.net") {
   }
-  .central-textlogo__image, .unicode.haudio a img {
+  .central-textlogo__image, .unicode.haudio a img,
+  .mw-ui-icon-popups-close::before {
```
Single colon as we do with other similar entries. =)

Aced14 replied:
Wasn't sure since it seemed like a mix of both one and two colon selectors were already in use for pseudo-elements. ``::selection`` and ``::-moz-selection`` are using two.

Single-colon syntax for pseudo-elements is also [deprecated in CSS3](https://www.w3.org/TR/selectors/#pseudo-element-syntax).

the-j0k3r commented on wikipedia-dark.user.css:
```diff
@@ -463,6 +467,13 @@ domain("wiki.rpcs3.net") {
+    background-color: rgba(34, 34, 34, 0.75) !important;
+  }
+  #mwe-popups-settings {
+    box-shadow: 0 2px 2px 0 rgba(85, 85, 85, 0.25) !important;
```
We dont prefix with the zero.

Aced14 replied:
Fixed :).

Aced14 commented:
@the-j0k3r I've updated my commit to address your line comments and removed the reference to #61 from the OP (since it's no longer relevant).

Haven't had a chance to look into brightening-up the text yet. Will try to look into it within the coming days.

Aced14 commented:
@the-j0k3r Updated again and revised the OP's after screenshot. Went with ``#ddd`` in the end.

Also applied it to a paragraph that appears in the popup if the user disables the setting and saves. Surprisingly, that paragraph's colour wasn't declared at all in Wikipedia's normal CSS and wasn't inherited from anything.

Aced14 commented:
One more force-push... Partially-inverted the disable notice's sample image to blend in better with its surroundings. Also added comparison screenshots of it to the OP.

the-j0k3r commented:
Hello again ;), this turned into a wack-a-mole game, this tends to happen when multiple PR's happen to same source file. To avoid the wack-a-mole rebase and fix game in future, try to group fixes in same PR but under different commits correspondingly named.

Now for this, If you could rebase and solve the conflicts we can get this merged asap and do a release =)

Thanks in advance.

Aced14 commented:
Just rebased this PR :).

Regarding grouping commits in the same PR, I usually try to separate my PRs so that each one is dedicated to a specific kind of change. That way, I can focus entirely on what's needed for one kind of change and easily force-push replacement commits as needed.

In any case, I don't think my future PRs will be as likely to run into conflicts as the ones I recently sent in. So hopefully things will be easier to deal with next time around.

the-j0k3r commented:
Thanks for your contribution, again =)
